deploy/docker/crawler_pool.py

```
@@ -15,35 +14,88 @@
+            try:
+                crawler = AsyncWebCrawler(
+                    config=cfg, thread_safe=False, crawler_strategy=crawler_strategy
+                )
+                await crawler.start()
+                async with LOCK:
+                    POOL[sig] = crawler
+                    LAST_USED[sig] = time.time()
+                fut.set_result(crawler)
+            except Exception as e:
+                fut.set_exception(e)
+                raise RuntimeError("Failed to start browser") from e
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)